web/packages/teleterm/src/ui/DocumentGateway/useDocumentGateway.ts

```diff
@@ -134,7 +122,7 @@ export default function useGateway(doc: types.DocumentGateway) {
     defaultPort,
     disconnect,
     connected,
-    reconnect,
+    reconnect: createGateway,
```

Comment:
Technically we could just export `createGateway` instead but the name `reconnect` makes sense in the context of `OfflineDocumentGateway`. `Offline/OnlineDocumentGateway` components are tightly bound to the hook so I didn't want to deal with renaming all of this right now.